### PR TITLE
fix: DetailsColumn tooltip shows on focus, is associated via aria-describedby

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

We provided an out-of-the-box method to create a DetailsColumn tooltip, but that tooltip wasn't keyboard accessible or associated via `aria-describedby` (with the exception of icon columns using `column.ariaLabel`).

## New Behavior

Now, when using `onRenderColumnHeaderTooltip`, the tooltip shows/hides on column header focus and blur, and is associated via `aria-describedby` unless the tooltip is already using the `column.ariaLabel` text since we don't want the same text as both label & description.

The reason to add `onFocus` and `onBlur` events to the columnheader node + call `.show()` and `.dismiss()` was to avoid needing to rearrange the DOM to make the TooltipHost the parent of the focusable columnheader.

## Related Issue(s)

- Fixes [16455](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/16455)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5a4b16715e8e929f11d8113f710e578ca73acaa6'>5a4b16715e8e929f11d8113f710e578ca73acaa6</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/BaseButton-mount_base.html">638</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/BaseButton-mount.html">629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Breadcrumb-mount_base.html">1663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Breadcrumb-mount.html">1682</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Checkbox-mount_base.html">1660</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Checkbox-mount.html">1675</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/CheckboxBase-mount_base.html">1472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/CheckboxBase-mount.html">1473</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ChoiceGroup-mount_base.html">2896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ChoiceGroup-mount.html">2898</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ComboBox-mount_base.html">654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ComboBox-mount.html">669</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/CommandBar-mount_base.html">6248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/CommandBar-mount.html">6282</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ContextualMenu-mount_base.html">13274</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ContextualMenu-mount.html">14277</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DefaultButton-mount_base.html">742</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DefaultButton-mount.html">764</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRow-mount_base.html">2186</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRow-mount.html">2159</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRowFast-mount_base.html">2175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRowFast-mount.html">2235</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRowNoStyles-mount_base.html">1991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DetailsRowNoStyles-mount.html">1999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Dialog-mount_base.html">2605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Dialog-mount.html">2605</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DocumentCardTitle-mount_base.html">228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/DocumentCardTitle-mount.html">221</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Dropdown-mount_base.html">1941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Dropdown-mount.html">1973</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/FocusTrapZone-mount_base.html">1128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/FocusTrapZone-mount.html">1120</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/FocusZone-mount_base.html">1061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/FocusZone-mount.html">1067</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-mount_base.html">41096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-mount.html">41136</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-virtual-rerender_base.html">19645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-virtual-rerender.html">19898</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedList-virtual-rerender-with-unmount.html">50431</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-mount_base.html">230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-mount.html">227</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-virtual-rerender_base.html">208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-virtual-rerender.html">208</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">219</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">222</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/IconButton-mount_base.html">1071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/IconButton-mount.html">1119</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Label-mount_base.html">323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Label-mount.html">325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Layer-mount_base.html">2669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Layer-mount.html">2661</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Link-mount_base.html">378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Link-mount.html">386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/MenuButton-mount_base.html">954</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/MenuButton-mount.html">947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/MessageBar-mount_base.html">21413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/MessageBar-mount.html">21383</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Nav-mount_base.html">1899</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Nav-mount.html">1950</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/OverflowSet-mount_base.html">758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/OverflowSet-mount.html">778</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Panel-mount_base.html">1761</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Panel-mount.html">1766</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Persona-mount_base.html">722</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Persona-mount.html">748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Pivot-mount_base.html">875</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Pivot-mount.html">869</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/PrimaryButton-mount_base.html">853</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/PrimaryButton-mount.html">831</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Rating-mount_base.html">4711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Rating-mount.html">4533</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SearchBox-mount_base.html">906</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SearchBox-mount.html">912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Shimmer-mount_base.html">1839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Shimmer-mount.html">1900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Slider-mount_base.html">1329</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Slider-mount.html">1292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SpinButton-mount_base.html">2896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SpinButton-mount.html">2852</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Spinner-mount_base.html">396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Spinner-mount.html">398</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SplitButton-mount_base.html">1810</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SplitButton-mount.html">1802</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Stack-mount_base.html">412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Stack-mount.html">397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/StackWithIntrinsicChildren-mount_base.html">829</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/StackWithIntrinsicChildren-mount.html">843</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/StackWithTextChildren-mount_base.html">2642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/StackWithTextChildren-mount.html">2640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SwatchColorPicker-mount_base.html">6030</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/SwatchColorPicker-mount.html">6171</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/TagPicker-mount_base.html">1479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/TagPicker-mount.html">1412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Text-mount_base.html">371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Text-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/TextField-mount_base.html">930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/TextField-mount.html">900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-mount_base.html">841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-mount.html">855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-virtual-rerender_base.html">589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-virtual-rerender.html">586</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1273</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Toggle-mount_base.html">602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/Toggle-mount.html">599</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/buttonNative-mount_base.html">187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28604/perf-test/buttonNative-mount.html">182</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/DetailsList/DetailsColumn.test.tsx [hunk 1]
```diff
@@ -184,11 +186,45 @@ describe('DetailsColumn', () => {
     expect(component.exists(`#${referenceId}`)).toBe(true);
   });
 
-  it('does not render invalid aria-describedby if custom DetailsHeader has onRenderColumnHeaderTooltip', () => {
+  it('does render invalid aria-describedby pointing to a custom tooltip not using ariaLabel', () => {
```
Comment (spmonahan) [inline packages/react/src/components/DetailsList/DetailsColumn.test.tsx:189]: Is this description accurate?

```suggestion
  it('renders valid aria-describedby pointing to a custom tooltip not using ariaLabel', () => {
```

File: packages/react/src/components/DetailsList/DetailsColumn.base.tsx [hunk 2]
```diff
@@ -127,6 +130,8 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
           }}
           data-automationid={'ColumnsHeaderColumn'}
           data-item-key={column.key}
+          onBlur={this._onColumnBlur}
```
Comment (spmonahan) [inline packages/react/src/components/DetailsList/DetailsColumn.base.tsx:133]: Does the tooltip need to appear on hover as well?

Comment (smhigley) [reply packages/react/src/components/DetailsList/DetailsColumn.base.tsx:133]: it already does; the TooltipHost is wrapped around the column header's contents with no padding in between, so the mouse target works. The issue is that it doesn't include the focus target, which is the root node of the DetailsColumn component.

**Review by spmonahan — APPROVED** [review]

**Pull request merged**